### DISCUSSION TITLE
FAILED: parsing YAML file snmp.yml: yaml: unmarshal errors:   line 2: field auths not found in type config.plain   line 15: field modules not found in type config.plain

### DISCUSSION THREAD

**Discussion by Reiswaffle** [prometheus/snmp_exporter#998] [Q&A]
Hi all, 
im currently trying to set up our fortigate firewall with SNMPv3, password and username but it seems like im too noob for this.

It currently already runs with SNMPv2 HPE Switches and works as intendet but when i try to import the fortigate prometheus looses connection to all hosts. 
YML file is below but i changed the password and username for obvious reasons:

[snmp.yml.txt](https://github.com/prometheus/snmp_exporter/files/12584413/snmp.yml.txt)

If i have any flair/tag mistakes please excuse me as this is my first time posting on a github Q&A

My System is Ubuntu server 22.04.3 LTS
SNMP Exporter Version: 0.23.0

**Comment by alexhatzo** [comment]
Did you figure this out? I am having a similar issue
